homeassistant/components/unifi/sensor.py

```diff
@@ -140,8 +143,10 @@ def name(self) -> str:
         return f"{super().name} {self.TYPE.capitalize()}"
 
     @property
-    def state(self) -> int:
+    def state(self) -> datetime:
         """Return the uptime of the client."""
+        if self.client.uptime < 1000000000:
```

Comment:
Smooth 😉

Reply:
I just need to differentiate on epoch time and not. I didn't look up any suggested method for it :)

Reply:
Love these kind of logical compatibility implementations, so that it is unrelated which version of unifi controller software is on the other end, it will just work :-) And the risk is very low that the uptime in seconds will pass this mark, but who knows 😉  

How long does it normally take that this fix will land in stable?

Reply:
This will probably not make it until the release 2021.04 due to the major changes of the tests